src/plots/wheelPlot.ts

```diff
@@ -0,0 +1,222 @@
+    private static _OUTER_RADIUS_KEY = "outer-radius";
+    private static _START_ANGLE_KEY = "start-angle";
+    private static _END_ANGLE_KEY = "end-angle";
+
+    /**
+     * @constructor
+     */
+    constructor() {
+      super();
+      this.addClass("wheel-plot");
+    }
+
+    public computeLayout(origin?: Point, availableWidth?: number, availableHeight?: number) {
+      super.computeLayout(origin, availableWidth, availableHeight);
+      this._renderArea.attr("transform", "translate(" + this.width() / 2 + "," + this.height() / 2 + ")");
+      let radiusLimit = Math.min(this.width(), this.height()) / 2;
+      if (this.innerRadius() != null && this.innerRadius().scale != null) {
+        this.innerRadius().scale.range([0, radiusLimit]);
+      }
+      if (this.outerRadius() != null && this.outerRadius().scale != null) {
+        this.outerRadius().scale.range([0, radiusLimit]);
+      }
+      return this;
+    }
+
+    protected _createDrawer(dataset: Dataset) {
+      return new Plottable.Drawers.Arc(dataset);
+    }
+
+    public entities(datasets = this.datasets()): PlotEntity[] {
+      let entities = super.entities(datasets);
+      entities.forEach((entity) => {
+        entity.position.x += this.width() / 2;
+        entity.position.y += this.height() / 2;
+      });
+      return entities;
+    }
+
+    protected _getDataToDraw() {
+      let dataToDraw = super._getDataToDraw();
+      if (this.datasets().length === 0) { return dataToDraw; }
+      let startAngleAccessor = Plot._scaledAccessor(this.startAngle());
+      let endAngleAccessor = Plot._scaledAccessor(this.endAngle());
+      let innerRadiusAccessor = Plot._scaledAccessor(this.innerRadius());
+      let outerRadiusAccessor = Plot._scaledAccessor(this.outerRadius());
+      let ds = this.datasets()[0];
+      let data = dataToDraw.get(ds);
+      let filteredData = data.filter((d, i) =>
+        Plottable.Utils.Math.isValidNumber(startAngleAccessor(d, i, ds)) &&
+        Plottable.Utils.Math.isValidNumber(endAngleAccessor(d, i, ds)) &&
+        Plottable.Utils.Math.isValidNumber(innerRadiusAccessor(d, i, ds)) &&
+        Plottable.Utils.Math.isValidNumber(outerRadiusAccessor(d, i, ds)));
+      dataToDraw.set(ds, filteredData);
+      return dataToDraw;
+    }
+
+    protected _propertyProjectors(): AttributeToProjector {
+      let attrToProjector = super._propertyProjectors();
+      let innerRadiusAccessor = Plot._scaledAccessor(this.innerRadius());
+      let outerRadiusAccessor = Plot._scaledAccessor(this.outerRadius());
+      let startAngleAccessor = Plot._scaledAccessor(this.startAngle());
+      let endAngleAccessor = Plot._scaledAccessor(this.endAngle());
+      attrToProjector["d"] = (datum: any, index: number, ds: Dataset) => {
+        let startAngle = startAngleAccessor(datum, index, ds) % (Math.PI * 2);
+        let endAngle = endAngleAccessor(datum, index, ds) % (Math.PI * 2);
+        while (endAngle < startAngle) { endAngle += Math.PI * 2; }
```

Comment:
I think the logic here is incorrect. I would expect something with a `startAngle()` of 0 and an `endAngle()` of 2π to draw a full disk but instead nothing is drawn: http://jsfiddle.net/71xu10uz/